mobly/test_runner.py

```diff
@@ -355,8 +355,8 @@ def get_info(objects):
             self.results.add_controller_info(module_config_name,
                                              controller_info)
         else:
-            self.log.warning("No controller info obtained for %s",
-                             module_config_name)
+            self.log.warning(("No optional debug info found for controller %s, "
```

Comment:
This still isn't that clear. Can you mention that the controller module doesn't implement get_info?